src/Call.php

```diff
@@ -53,14 +51,14 @@ class Call
     /**
      * @param string $method
      * @param string $decodeType
-     * @param Message $message
+     * @param mixed $message
```

Comment:
WDYT  about `Message|mixed`? Using [this](https://github.com/googleapis/gax-php/blob/master/src/ServerStream.php#L88) as precedent.

Reply:
Agreed, done.